src/Tests/CaptchaAdminTestCase.php

```diff
@@ -42,50 +41,47 @@ public function testCaptchaPointSettingGetterAndSetter() {
     captcha_set_form_id_setting($comment_form_id, 'none');
     /* @var CaptchaPoint $result */
     $result = captcha_get_form_id_setting($comment_form_id);
-    $this->assertNotNull($result, 'Setting and getting CAPTCHA point: none', 'CAPTCHA');
-    // $this->assertNull($result->module, 'Setting and getting
-    // CAPTCHA point: none', 'CAPTCHA');
-    $this->assertNull($result->getCaptchaType(), 'Setting and getting CAPTCHA point: none', 'CAPTCHA');
+    $this->assertNotNull($result, 'CAPTCHA exists', 'CAPTCHA');
+    $this->assertEqual($result->getCaptchaType(), 'none', 'CAPTCHA type: none', 'CAPTCHA');
     $result = captcha_get_form_id_setting($comment_form_id, TRUE);
+    $this->assertNotNull($result, 'CAPTCHA exists', 'CAPTCHA');
     $this->assertEqual($result, 'none', 'Setting and symbolic getting CAPTCHA point: "none"', 'CAPTCHA');
```

Comment:
$result->module doesn't exist at all anymore. The symbolic part is a little unclear now and hard to test, all old tests for it seem broken.